tests/BasicSearchTests.FunctionalTests.Core/V2SearchFunctionalTests.cs

```diff
@@ -37,5 +38,30 @@ public class V2SearchFunctionalTests : BaseFunctionalTests
             Assert.True(result.TotalHits.HasValue && result.TotalHits.Value > 0, "No results found, should find atleast some results for empty string query.");
             Assert.NotNull(result.Data);
         }
+
+        [Fact]
+        public async Task EmptyQueryResultHaveRequiredProperties()
+        {
+            // Act
+            var response = await Client.GetAsync(new V2SearchBuilder().RequestUri);
+            var content = await response.Content.ReadAsStringAsync();
+
+            // Newtonsoft's Json deserialization will throw exceptions if required
+            // properties are missing.
+            JsonConvert.DeserializeObject<V2SearchResult>(content);
```

Comment:
This test should not throw an exception on the happy path. If the required fields are not present, an exception will be thrown and will cause the unit test to fail.